metadata-service/restli-api/src/main/snapshot/com.linkedin.lineage.relationships.snapshot.json

```diff
@@ -96,6 +96,11 @@
       "name" : "total",
       "type" : "int",
       "doc" : "Total number of results in the result set"
+    }, {
+      "name" : "filtered",
```

Comment:
not a blocker: but we might need further breakdowns in the future to distinguish between the states of the entities -> non-existent, soft-deleted etc.

Reply:
Sounds good - I agree. This would then become "total filtered". In that case, we can extend to add

`filteredSoftDeleted`

and `filteredNonExistent` 

If the client wants to distinguish.

The other option I'm thinking is letting the client actually pass an argument which dictates _how_ we do post-filtering on the result set.

You can imagine something like:

```
getEntityLineage(input: { ... filterMode: FilterMode.DEFAULT / FilterMode.SOFT_DELETED / FilterMode.NON_EXISTENT }) 
```